### PR TITLE
feat: [CCH-41] congregation binding and ui tweaks

### PR DESCRIPTION
**Pull request by JudeDavis1**
https://www.notion.so/Bind-congregation-to-user-6ac372609725404f9924e234b9bd3409?pvs=4

### CODE REVIEW

File: packages/frontend/app/dashboard/components/admin-dashboard.tsx [hunk 1]
```diff
@@ -11,45 +11,41 @@ import {
...
-        email: "example@gmail.com",
-        congregation: "London, Uxbridge",
-      }}
-    />,
-    <MeetingDutiesCard data={placeholderDuties} />,
-    <InformationCard data={placeholderInformationBoard} />,
-    <CongEventsCard data={placeholderCongEvents} />,
-    <PublicWitnessingCard />,
+  const userDashboardItems: DashboardItem[] = [
+    () => (
```
Comment (j-koziel) [inline packages/frontend/app/dashboard/components/admin-dashboard.tsx:37]: What's the difference here?

**Review by j-koziel — APPROVED** [review]
LGTM, just a small question :)

File: packages/frontend/app/dashboard/components/admin-dashboard.tsx (hunk 1 above)
Comment (JudeDavis1) [reply packages/frontend/app/dashboard/components/admin-dashboard.tsx:37]: This is called lazy evaluation. I didn't want the components to be created all at once on every render, instead just functions to create them. This allows them to be created dynamically. Probably a negligible effect but just didn't want to use up too much memory

Comment (JudeDavis1) [reply packages/frontend/app/dashboard/components/admin-dashboard.tsx:37]: Especially since components are quite heavy in general already

**Pull request merged**